### PR TITLE
Cherrypicks from master

### PR DESCRIPTION
**Pull request by ostromart**
add readable ut. (https://github.com/istio/operator/pull/187)
Eliminate use of vfsgen (https://github.com/istio/operator/pull/180)
Revert "Eliminate use of vfsgen (#180)" (https://github.com/istio/operator/pull/191)
Remove charts (https://github.com/istio/operator/pull/190)
fix crds chart directory. (https://github.com/istio/operator/pull/192)
k8s resource limits cannot be lower than requests (https://github.com/istio/operator/pull/194)
fix mandiff gaps (https://github.com/istio/operator/pull/193)
Update the README file (https://github.com/istio/operator/pull/178)
Modify output path for charts (https://github.com/istio/operator/pull/195)
Set dry run default to false (https://github.com/istio/operator/pull/197)
Fix update chart script (https://github.com/istio/operator/pull/196)
Update README for dry-run off by default (https://github.com/istio/operator/pull/199)
Fix translate and profile issues (https://github.com/istio/operator/pull/198)
fix broken manifest generating and mandiff. (https://github.com/istio/operator/pull/201)

### CODE REVIEW

**Review by rlenglet — CHANGES_REQUESTED** [review]
Remove the `Add support for installation from url #186` commit form this PR. This is a new feature, and there was contention in review comments. It's not clear to me that there is consensus that this should be included in 1.3.
Please submit that commit as a separate PR so we can discuss its backport individually.
The other commits LGTM.

File: installer.sha [hunk 1]
```diff
@@ -0,0 +1 @@
+228635e5b87925773b9b4e12ceb89ec8cecadab3
```
Comment (rlenglet) [inline installer.sha]: Fix this SHA. This commit is in the `master` branch, and not in the `release-1.3` branch.
You must use a SHA in the `release-1.3` branch here.

**Review by rlenglet — CHANGES_REQUESTED** [review]
Fix the istio/installer SHA.

File: installer.sha (hunk 1 above)
Comment (ostromart) [reply installer.sha]: Done.

**Review by rlenglet — APPROVED** [review]

**Pull request merged**